IntelliTectAnalyzer/IntelliTectAnalyzer/Analyzers/AttributesOnSeparateLines.cs

```diff
@@ -38,6 +38,7 @@ private static void AnalyzeSymbol(SymbolAnalysisContext context)
             if (namedTypeSymbol.GetAttributes().Any())
             {
                 IDictionary<int, AttributeData> lineDict = new Dictionary<int, AttributeData>();
+                lineDict.Add(namedTypeSymbol.Locations[0].GetLineSpan().StartLinePosition.Line, null);
```

Comment:
```suggestion
                lineDict[namedTypeSymbol.Locations[0].GetLineSpan().StartLinePosition.Line] = null;
```

Though pretty unlikely, I believe this throws a duplicate key exception if you have two properties declared on the same line with an attribute above them.

Reply:
Thanks @Keboo good catch. I just changed that bit of code because I felt like I was misusing the dictionary from its intended purpose.